File: exp2_Fisica.py
from math import sin,cos, pi

def rad(angle):
    return (angle/180)*pi

def main():
    T2 = 0
    T1 = 2.934

    for i in range(1, 40):
        angleT2 = rad(i)
        angleT1 = rad(140)

        T2 = (T1*sin(angleT1)-3.4335)/sin(angleT2)

        if(T1*sin(angleT1)-3.4335+T2*sin(angleT2) == 0):
            print(f'\nEl valor de T2 es: {T2} en el angulo {angleT2}')
            print(f'La sumatoria en X es: {T1*cos(angleT1)+T2*cos(angleT2)}')
            print(f'La sumatoria en Y es: {T1*sin(angleT1)-3.4335+T2*sin(angleT2)}')
    
        

main()

File: exp2_Fisica.cpp
#include<iostream>
#include<math.h>

using namespace std;

int main(){

    float T2;   
    float PI = 3.14159;


    for(int i=20;i<40;i++){
        long double angle = i*PI/180;
        T2 = 1.54177665/sin(angle);

        cout<<"El angulo es: "<<i<<" en rad: "<<angle<<endl;
        cout<<"El valor de T2 es: "<<T2<<endl;
        cout<<"La sumatoria de tensiones en Y es: "<<(1.891723935+T2-3.4335)<<endl;
    }

    cout<<20/180;

    return 0;
}